plugins/plugin_loader.cpp

```diff
@@ -52,34 +64,42 @@ void PluginLoader::LoadPlugins(const std::filesystem::path& plugins_dir_path)
             continue;
         }
 
-        NativeLibraryHandle handle = m_library_loader->Load(file_path);
-        if (handle == nullptr)
+        absl::StatusOr<NativeLibraryHandle> handle_status = m_library_loader->Load(file_path);
```

Comment:
nit: abseil recommends treating StatusOr like a smart pointer, so their recommendation for naming is to just call it the thing without a suffix like `_or` or, in this case, `_status`. https://abseil.io/tips/181#avoiding-the-_or-suffix
```suggestion
        absl::StatusOr<NativeLibraryHandle> handle = m_library_loader->Load(file_path);
```

Reply:
fixed